eng/build.yml

```diff
@@ -0,0 +1,83 @@
+parameters:
+  agentOs: ''
+  queue: {}
+  variables: {}
+
+phases:
+- template: /eng/common/templates/phases/base.yml
+  parameters:
+    name: ${{ parameters.agentOs }}
+    enableTelemetry: true
+    enableMicrobuild: true
```

Comment:
this looks like it's for public builds only (ie, I don't see any reference to an internal windows build queue).  Is this intended to be just for public builds (if so, then this should just be omitted)

Reply:
Will remove it.